### PR TITLE
VectorDraw Enhancements

### PR DESCRIPTION
**Pull request by mtyaka**
Changes in this PR:

* It is now possible to start drawing a new vector when hovering over a label.
* Disable info box with coordinates.
* Ability to only grade x or y coordinate for both tail and tip positions.
* Ability to specify custom checks in problem definition (requires that the custom check is implemented in python code in the xml file).
* Provide custom grading for the energy levels example.
* Update the README file with more details.

### CODE REVIEW

**Comment by mtyaka** [comment]
@dseaton This PR includes most changes requested in https://github.com/open-craft/jsinput-vectordraw/pull/6#issue-67802305. I updated the energy level example at https://studio.edge.edx.org/container/i4x://DavidsonNext/Ann101/vertical/508d02e23a1e4d2bab6c00643424f228 with some custom grading functions.

One change that isn't included in this PR is the ability to set up problems with unlabeled vectors. Current code assumes vectors have labels/identifiers in multiple places and removing the identifiers is going to cause problems with grading. I propose we try to compose a list of example exercises where labels aren't needed, so that we can come up with a generic solution that would work for most cases.

**Comment by dseaton** [comment]
@mtyaka This is great! Just to confirm, all examples are working for me.

Regarding unlabeled vectors, I'll do the following:
1. Create one or two unlabeled examples.
2. We will ask our AP instructors to give us feedback on the tool, and filter out the unlabeled requests to help inform the coding.

Does that sound reasonable? @mtyaka @antoviaque 

UGH - I wrote this about 7 hours ago, but forgot to hit comment. My sincere apologies.

**Comment by mtyaka** [comment]
@dseaton That sounds good, thanks.

File: api-example.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <html>
     <head>
         <link href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
-        <link href="/c4x/edX/JS101/asset/vectordraw.css" rel="stylesheet" type="text/css" />   <!-- Default styles can be overriden here -->
+        <link href="/c4x/edX/JS101/asset/vectordraw.css" rel="stylesheet" type="text/css" />
```
Comment (e-kolpakov) [inline api-example.html]: @mtyaka README talsk about `DavidsonNext/Ann101/`, and this file uses `edX/JS101` - might make sense to mention that too (or change to DavidsonNext as well)

**Comment by e-kolpakov** [comment]
@mtyaka a couple of issues:

* `api-example.html` still uses `simple_car.png`, but it is removed
* `api-example.xml` is not updated to have default height of 480px - scroll appears.

**Comment by dseaton** [comment]
@mtyaka Great. I also agree with @e-kolpakov regarding changing the file paths to reflect Davidson Next. Possible to use the /static/ folder? 

I'll work on the unlabeled vector examples.

**Comment by e-kolpakov** [comment]
@mtyaka :+1:

**Pull request merged**